### PR TITLE
Update German Translation

### PR DESCRIPTION
**Pull request by FiLeonard**
fixed some mistakes, translated a few new or changed strings, altered and enhanced a few translations and capitalised 'du'

### CODE REVIEW

**Comment by FiLeonard** [comment]
fixed mistakes, translated a few new or changed strings, altered and enhanced some translations and capitalised 'du'

**Comment by pgert** [comment]
Is it really that difficult to find a german equivalent to "savestate"?
I use "sparpunkt" for my swedish translation, which should apply to german as well, yes?

File: locales/de_DE/pcsx2_Iconized.po [hunk 1]
```diff
@@ -402,66 +417,77 @@ msgid ""
...
+" * Wizardry"
 
 #: pcsx2/gui/Panels/GameFixesPanel.cpp:83
 msgid ""
 "Known to affect following games:\n"
 " * Mana Khemia 1 (Going \"off campus\")\n"
-msgstr "DMA Busy hack\n"
+msgstr ""
+"Bekannte Effekte auf folgende Spiele:\n"
+" * Mana Khemia 1 (Going \"off campus\")\n"
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: "Going off campus" scheint nicht der Titel des Spiels zu sein, sondern die Beschreibung, wo der Fehler auftritt. Müsste also übersetzt werden. Wenigstens das "Going".

File: locales/de_DE/pcsx2_Iconized.po [hunk 2]
```diff
@@ -402,66 +417,77 @@ msgid ""
...
 
 #: pcsx2/gui/Panels/GameFixesPanel.cpp:88
 msgid ""
 "Known to affect following games:\n"
 " * Test Drive Unlimited\n"
 " * Transformers"
-msgstr "VIF1 Fifo hack."
+msgstr ""
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: Zeilensprung?

File: locales/de_DE/pcsx2_Iconized.po [hunk 3]
```diff
@@ -402,66 +417,77 @@ msgid ""
...
 "It's better to enable 'Automatic game fixes' at the main menu instead, and "
 "leave this page empty. ('Automatic' means: selectively use specific tested "
 "fixes for specific games). Manual game fixes will NOT increase your "
 "performance. In fact they may decrease it."
 msgstr ""
-"Spielefixes können spezifische, bekannte Fehler in einigen Spielen beheben. "
-"Sie sollten aber nur für diese Spiele aktiviert werden."
+"Es ist besser, automatische Spielefixes im Hauptmenü zu aktivieren, und "
+"diese Seite frei zu lassen. (Automatisch heißt, dass selektiv spezifische "
+"Spielefixes angewendet werden). Manuelle Spielefixes werden die Fps-Rate "
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: FPS groß?

File: locales/de_DE/pcsx2_Iconized.po [hunk 4]
```diff
@@ -540,87 +567,103 @@ msgstr ""
 "Abbrechen."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:29
-#, fuzzy
 msgid ""
 "-3 - Reduces the EE's cyclerate to about 50%.  Big speedup, but *will* cause "
 "stuttering audio on many FMVs."
-msgstr "Stufe 3."
+msgstr ""
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: Zeilensprung?

File: locales/de_DE/pcsx2_Iconized.po [hunk 5]
```diff
@@ -540,87 +567,103 @@ msgstr ""
...
+"-2 - Reduziert die Zyklusrate des EE auf ungefähr 60%.  Moderater FPS-"
+"Zuwachs, aber wird in vielen FMVs Stottern verursachen."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:39
-#, fuzzy
 msgid ""
 "-1 - Reduces the EE's cyclerate to about 75%.  Mild speedup for most games "
 "with high compatibility."
-msgstr "Stufe 2."
+msgstr ""
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: Zeilensprung?

File: locales/de_DE/pcsx2_Iconized.po [hunk 6]
```diff
@@ -540,87 +567,103 @@ msgstr ""
...
+"für viele Spiele bei hoher Kompatibilität."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:45
-#, fuzzy
 msgid ""
 "0 - Default cyclerate (100%). This closely matches the actual speed of a "
 "real PS2 EmotionEngine."
-msgstr "Deaktiviert"
+msgstr ""
+"0 - Standart-Zyklusrate (100%). Dies entspricht sehr nah der originalen PS2 "
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: nah? Vielleicht "in etwa" oder "ungefähr"? Oder "Es kommt [...] nah"? Aber klingt seltsam.

File: locales/de_DE/pcsx2_Iconized.po [hunk 7]
```diff
@@ -540,87 +567,103 @@ msgstr ""
...
+"Hardwareanforderungen. Kann merklich die In-Game FPS erhöhen."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:60
 msgid ""
 "3 - Increases the EE's cyclerate to about 300%. Greatly increases hardware "
 "requirements, may noticeably increase in-game FPS.\n"
 "This setting can cause games to FAIL TO BOOT."
 msgstr ""
+"3 - Erhöht die Zyklusrate des EE auf ungefähr 300%. Erhöht die "
+"Hardwareanforderungen deutlich. Kann deutlich die In-Game FPS erhöhen.Diese "
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: Leerzeichen nach dem Punkt nach "erhöhen" fehlt.

File: locales/de_DE/pcsx2_Iconized.po [hunk 8]
```diff
@@ -540,87 +567,103 @@ msgstr ""
...
+"merklicher Geschwindigkeitszuwachs in einigen Spielen."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:93
 msgid ""
 "3 - Maximum VU Cycle Stealing.  Usefulness is limited, as this will cause "
 "flickering visuals or slowdown in most games."
-msgstr "Stufe 3."
+msgstr ""
+"3 - Maximales Stehlen von VU-Zyklen. Eingeschränkte Brauchbarkeit, da in den "
+"meisten Spielen visuelles Flackern oder Abbremsen des Spiels entsteht."
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: Vielleicht lieber "ein Abbremsen" und vielleicht lieber "auftreten" oder "bemerkbar sind"?

File: locales/de_DE/pcsx2_Iconized.po [hunk 9]
```diff
@@ -540,87 +567,103 @@ msgstr ""
...
 "Speedhacks usually improve emulation speed, but can cause glitches, broken "
 "audio, and false FPS readings.  When having emulation problems, disable this "
 "panel first."
 msgstr ""
-"Speedhacks verbessern die FPS Leistung der Emulation auf deinem PC indem sie "
-"Abkürzungen nehmen oder die PS2 Hardware untertakten. Sie können "
-"Emulationsprobleme auslösen, daher empfehlen wir bei Fehlern zuerst die "
+"Speedhacks verbessern die FPS Leistung der Emulation auf Deinem PC, indem "
+"sie Abkürzungen nehmen oder die PS2 Hardware untertakten. Sie können "
+"Emulationsprobleme auslösen, daher empfehlen wir, bei Fehlern zuerst die "
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: Nach auslösen müsste vermutlich sogar ein Punkt.

File: locales/de_DE/pcsx2_Iconized.po [hunk 10]
```diff
@@ -646,7 +691,7 @@ msgid ""
 "Updates Status Flags only on blocks which will read them, instead of all the "
 "time. This is safe most of the time, and Super VU does something similar by "
 "default."
-msgstr "Lässt einige VU Statusflags aus. Sicher."
+msgstr "Lässt einige VU Statusflags aus, um sie in Blöcken zu lesen. Sicher."
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: Nicht deine Übersetzung aber das "Sicher" sieht hier seltsam aus. Im Original ist eine Bemerkung zu SuperVU.

File: locales/de_DE/pcsx2_Iconized.po [hunk 11]
```diff
@@ -704,7 +749,9 @@ msgstr ""
 msgid ""
 "Enable this if you think MTGS thread sync is causing crashes or graphical "
 "errors."
-msgstr "Nur für das Debugging aktivieren. Sehr langsam."
+msgstr ""
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: Zeilensprung?

File: locales/de_DE/pcsx2_Iconized.po [hunk 12]
```diff
@@ -734,8 +781,9 @@ msgid ""
 "not a critical error, since the sVU rec is obsolete, and you should use "
 "microVU instead anyway. :)"
 msgstr ""
-"Der SuperVU Recompiler konnte nicht genügend virtuellen Speicher allokieren. "
-"Versuche es mit microVU!"
+"Der SuperVU Recompiler konnte nicht genügend virtuellen Speicher "
+"alloziieren. Dies ist kein kritischer Fehler, da der sVU rec obsolet ist. Du "
```
Comment (willkuer) [inline locales/de_DE/pcsx2_Iconized.po]: allozieren mit einem 'i'

**Comment by willkuer** [comment]
> Is it really that difficult to find a german equivalent to "savestate"?
> I use "sparpunkt" for my swedish translation, which should apply to german as well, yes?

Maybe "Speicherslot". Sounds awful though. I wouldn't translate every English word. If the English word makes more sense I'd just keep it. We also don't translate "internet", "mail" and stuff like that.

Also "Sparpunkt" has no or the wrong meaning. "Save" can not be translated by "spar" but should be translated by "Speicher" and I have no idea how to directly translate "state".

**Comment by FiLeonard** [comment]
'Speicherstand' would be correct but it would be a translation for save and savestate.  'Speicherzustand' sounds wrong. I would probably prefer 'Speicherpunkt' versus 'Speicherstand' even if 'Speicherstand' is not used in the UI. Then again 'Speicherpunkte' are often used inside of games. It's hard to find something that's not ambiguous. Maybe the original term is the best.

**Comment by willkuer** [comment]
"Speicherstand" sounds quite good but I've never seen it being used before.

**Comment by FiLeonard** [comment]
It is common for saves actually. At least it used to be. Nowadays the english term is often used. Maybe we could use 'Speicherstand'. Virtualbox uses 'Sicherungspunkt'. Maybe 'Sicherungsstand' or 'Sicherungspunkt' are also candidates.

**Comment by FiLeonard** [comment]
Actually i kind of like 'Sicherungsstand'

**Comment by ramapcsx2** [comment]
I'm leaning towards not translating save states at all, relying on the term being popularized by other emulators.
The problem with translating is that the final strings will get pretty large for use in the UI.
And then it will probably sound very strange as well :p

File: locales/de_DE/pcsx2_Iconized.po [hunk 13]
```diff
@@ -71,28 +74,28 @@ msgid ""
...
 msgid ""
 "PCSX2 requires a PS2 BIOS in order to run.  For legal reasons, you *must* "
 "obtain a BIOS from an actual PS2 unit that you own (borrowing doesn't "
 "count).  Please consult the FAQs and Guides for further instructions."
 msgstr ""
-"PCSX2 benötigt ein PS2 BIOS Abbild. Da Sony den BIOS Code lizenziert hat "
-"dürfen wir dieses nicht mit PCSX2 vertreiben. Du musst daher das BIOS Abbild "
-"deiner eigenen PS2 benutzen! Näheres dazu in der FAQ."
+"PCSX2 benötigt ein PS2 BIOS Abbild. Da Sony den BIOS Code lizenziert hat, "
+"dürfen wir dieses nicht mit PCSX2 vertreiben. du musst daher das BIOS Abbild "
```
Comment (ramapcsx2) [inline locales/de_DE/pcsx2_Iconized.po]: Das 'du' darf wieder groß bitte :)

**Comment by FiLeonard** [comment]
Well. I think i got it mostly.

**Comment by ramapcsx2** [comment]
If you want to change the BIOS license string, you're very welcome to do so.
Your replacement a) sounds better and b) is definitely more correct.
But yeah, this looks fine and I'll merge it if you want. Thanks! :)

**Comment by FiLeonard** [comment]
Well i basically wanted to translate all those untranslated bits and correct a few mistakes, of course embedding a few of my own ^^ I consider it done now if nothing is still wrong with it. Thanks for all the comments :-)

**Comment by ssakash** [comment]
@ramapcsx2 

I'd prefer if squash merge is done for trivial changes like this one, it pollutes the Git history when you merge 9 commits which have useless commit titles like ``small update``, ``corrections``, ``revert``etc.

Here's a picture of Git history for example:
gh

**Pull request merged**